tests/tests_integration/test_api/test_templates.py

```diff
@@ -251,6 +252,7 @@ def test_view_resolve(self, cognite_client, new_view):
         )
         assert res == expected
 
+    @pytest.mark.skip("Test data is missing, cannot be ported to another CDF Project")
```

Comment:
Ok, not maintaining? I do not know (and do not want to spend the time) digging into the missing data for a deprecated service.

Reply:
Feel free to remove all integration tests for Templates; afaik we don't even deploy the API to more than 1 or 2 clusters.

Reply:
Done